src/templates/default/reviews/review_stages.html

```diff
@@ -0,0 +1,141 @@
+		<button class="btn btn-natural-noborder hotkey" value="{{ rs }}">{{ rs }}</button>
+		{% endfor %}
+		<!--Review Change Setting-->
+		<form action="{% url 'review_stages' %}" method="post">
+			{% csrf_token %}
+			<div class="panel panel-default">
+				<div class="panel-heading">{% trans 'Review Stages' %}
+					<div class="material-switch pull-right">
+						<a href="https://github.com/pycontw/pycon.tw/blob/master/src/reviews/README.md">See Proposal Review System
+							Documentation</a>
+					</div>
+				</div>
+				<ul class="list-group">
+					<li class="panel-title list-group-item">
+						{{ ".proposals.creatable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.creatable" type='hidden' value="false"/>
+							<input id="proposals.creatable" name="proposals.creatable" type="checkbox" value="true"/>
+							<label for="proposals.creatable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.editable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.editable" type='hidden' value="false"/>
+							<input id="proposals.editable" name="proposals.editable" type="checkbox" value="true"/>
+							<label for="proposals.editable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.withdrawable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.withdrawable" type='hidden' value="false"/>
+							<input id="proposals.withdrawable" name="proposals.withdrawable" type="checkbox" value="true"/>
+							<label for="proposals.withdrawable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.visible.to.submitters"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="reviews.visible.to.submitters" type='hidden' value="false"/>
+							<input id="reviews.visible.to.submitters" name="reviews.visible.to.submitters"
+										 type="checkbox" value="true"/>
+							<label for="reviews.visible.to.submitters" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.stage"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input id="reviews.stage" name="reviews.stage" type="number" min="0" max="2"
+										 placeholder="Int" required="required">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.disable.after"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input id="proposals.disable.after" name="proposals.disable.after" type="datetime-local"
+										 placeholder="YYYY-mm-ddThh:mm">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						timezone
```

Comment:
What's the purpose of allowing updating the timezone? 
Aren't we switching the stages based AoE (anywhere on Earth) time? The README of the review stages does not mention timezone too.